--- /net/minecraft/inventory
+++ /net/minecraft/inventory
@@ -25,13 +25,12 @@
 import xyz.openmodloader.launcher.strippable.Side;
 import xyz.openmodloader.launcher.strippable.Strippable;
 
-public class ContainerRepair extends Container {
+public class ContainerRepair extends xyz.openmodloader.event.impl.player.EventTakeSlot.Repair.OMLContainerRepair {
     private static final Logger LOGGER = LogManager.getLogger();
     private final IInventory outputSlot;
     private final IInventory inputSlots;
     private final World theWorld;
     private final BlockPos selfPosition;
-    public int maximumCost;
     private int materialCost;
     private String repairedItemName;
     private final EntityPlayer thePlayer;
